screens/team-codes/CreateScreen.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import { StyleSheet, TextInput, View } from "react-native";
+import KitButtonSupreme from "../../components/KitButtonSupreme";
+import KitText from "../../components/KitText";
+import Colors from "../../constants/Colors";
+import FontStyles from "../../constants/FontStyles";
+import { createTeam } from "../../utils/db/teams";
+import KitBackgroundScreen from "../../components/KitBackgroundScreen";
+
+export default function Create(props) {
+  const [teamName, setTeamName] = useState("");
+  const [teamCode, setTeamCode] = useState("");
+
+  return (
+    <KitBackgroundScreen
+      title="Set Team Name:"
+      onPressBack={() => {
+        props.navigation.navigate("Teams");
+      }}
+    >
+      <View style={styles.textInputContainer}>
+        {/* Evan TODO: Make this input a separate compo */}
+        <TextInput
+          style={styles.textInput}
+          placeholder="Ex) Team One"
+          onChangeText={setTeamName}
+          value={teamName}
+        />
+        <KitButtonSupreme
+          onPress={async () => {
+            const teamCode = await createTeam(teamName, "TODO: add userid");
```

Comment:
Not a 100% but I think await is the right choice here, b/c we want to block the user from potentially copying a non-existent join code.